lib/git_reflow/git_helpers.rb

```diff
@@ -27,6 +27,19 @@ def current_branch
       run("git branch --no-color | grep '^\* ' | grep -v 'no branch' | sed 's/^* //g'", loud: false).strip
     end
 
+    def pull_request_template
```

Comment:
Can you create a spec for this?  You can stub reading of a file with `allow(File).to receive(:read).with(".github/PULL_REQUEST_TEMPLATE.md").and_return("Some awesome template content")`